src/vs/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -71,21 +206,104 @@ export class SelectBox extends Widget {
 		}));
 	}
 
+	private registerSelectDropDownListeners() {
+		// Have to implement both keyboard and mouse controllers to handle disabled options
+		// Intercept mouse events to override normal select actions on parents
+
+		this.toDispose.push(dom.addDisposableListener(this.selectElement, dom.EventType.CLICK, () => {
+			this.showSelectDropDown();
+			event.preventDefault();
```

Comment:
@cleidigh I suggest to use the shorter [`stop`](https://github.com/Microsoft/vscode/blob/master/src/vs/base/browser/dom.ts#L766) function with the option to cancel bubble.